packages/kbn-apm-synthtrace/src/scenarios/azure_functions.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { apm, ApmFields } from '@kbn/apm-synthtrace-client';
+import { Scenario } from '../cli/scenario';
+import { RunOptions } from '../cli/utils/parse_run_cli_flags';
+import { getSynthtraceEnvironment } from '../lib/utils/get_synthtrace_environment';
+
+const ENVIRONMENT = getSynthtraceEnvironment(__filename);
+
+const scenario: Scenario<ApmFields> = async (runOptions: RunOptions) => {
+  return {
+    generate: ({ range }) => {
+      const timestamps = range.ratePerMinute(180);
+
+      const cloudFields: ApmFields = {
+        'cloud.provider': 'azure',
+        'cloud.service.name': 'functions',
+        'cloud.region': 'Central US',
+      };
+
+      const instanceALambdaDotnet = apm
+        .serverlessFunction({
+          serviceName: 'azure-functions',
+          environment: ENVIRONMENT,
+          agentName: 'dotnet',
+          functionName: 'fn-dotnet-1',
+        })
```

Comment:
Shouldn't you add `serverlessType` here? and in the others `serverlessFunction`?

```suggestion
        .serverlessFunction({
          serviceName: 'azure-functions',
          environment: ENVIRONMENT,
          agentName: 'dotnet',
          functionName: 'fn-dotnet-1',
          serverlessType: 'azure.functions'
        })
```